### ISSUE TITLE
chloro persistence

### ISSUE THREAD

**Issue by Tyrsen** [littlesera/LSCG#219]
Even if you have "chloroform never fades" selected, it no longer functions after re-login.

**Comment by littlesera** [comment]
Could you provide a little more information? What specifically no longer functions?

**Comment by Tyrsen** [comment]
> Could you provide a little more information? What specifically no longer functions?

All of the effects of the chloroform are not there.
It's like if you didnt have the chloro cloth on you at all.

**Comment by littlesera** [comment]
hm, my test account seemed to still behave as if chloroformed. She can't access player settings (immersive), cannot walk, and is forced down. Do you mean the periodic event messages?

**Comment by Tyrsen** [comment]
No, all of it. I am using an ipad (ios safari browser) if that matters.

**Comment by littlesera** [comment]
Strange indeed! I'll try with that browser..
